AMA.iml

```diff
@@ -8,7 +8,7 @@
       <sourceFolder url="file://$MODULE_DIR$/src/main/resources" type="java-resource" />
       <excludeFolder url="file://$MODULE_DIR$/target" />
     </content>
-    <orderEntry type="jdk" jdkName="1.8" jdkType="JavaSDK" />
+    <orderEntry type="inheritedJdk" />
```

Comment:
WHATEVER YOU WANT IT TO DO BABY


I don't actually know. I think it has something to do with how intelliJ locates the JDK